### DISCUSSION TITLE
Lokinet flatpak/snap package?

### DISCUSSION THREAD

**Discussion by Darin755** [oxen-io/lokinet#1949] [Ideas]
Hello,
Is there any plans to create a version of lokinet in a portable application format? I am on Fedora and it would be nice to have a simple way of installing on any distro.

Darin755

**Comment by majestrate** [comment]
> Hello,
> Is there any plans to create a version of lokinet in a portable application format? I am on Fedora and it would be nice to have a simple way of installing on any distro.
> 
> Darin755

rpm.oxen.io

we have no plans for a flatpak or snap as neither is worth the effort

keep in mind, there is so such thing as "portable" packaging in userland linux.

**Reply by Darin755** [reply]
>  there is so such thing as "portable" packaging in userland linux.

Well that is technically true it is still more portable than a deb or rpm. Having a portable app format makes it easy to install and remove applications and allows for more flexability. I might look into creating a unofficial flatpak although it may require a substantial amount of effort to get the permissions right.

**Reply by majestrate** [reply]
flatpak is not really "portable", it is more like yet another package manager, just one that vibes a bit better with other package managers. i am rather skeptical that flatpak has what is needed to make lokinet work and you would unironically have a better time using lxc (or docker if you have masochistic tendencies).

**Reply by Darin755** [reply]
Your probably right. I just like that flatpaks are isolated from the rest of the system.
